kustomize/README.md

```diff
@@ -125,10 +53,63 @@ Alternatively, if you would like to quickly provision a GKE cluster, you can use
     shippingservice-6ccc89f8fd-v686r         1/1     Running   0          2m58s
     ```
 
+    _Note: It may take 2-3 minutes before the changes are reflected on the deployment._
+
 1. Access the web frontend in a browser using the frontend's `EXTERNAL_IP`.
 
     ```
     kubectl get service frontend-external | awk '{print $4}'
     ```
 
     Note: you may see `<pending>` while GCP provisions the load balancer. If this happens, wait a few minutes and re-run the command.
+
+## Deploy Online Boutique variations with Kustomize
+
+Here is the list of the variations available as Kustomize components that you could leverage:
+
+- [**Changing to the Cymbal Shops Branding**](components/cymbal-branding)
+  - Changes all Online Boutique-related branding to Google Cloud's fictitious company — Cymbal Shops. The code adds/enables an environment variable `CYMBAL_BRANDING` in the `frontend` service.
+- [**Integrating with Google Cloud Operations**](components/google-cloud-operations)
+  - Enables Monitoring (Stats), Tracing, Profiler, and Debugger for various services within Online Boutique. The code removes the existing environment variables (`DISABLE_STATS`, `DISABLE_TRACING`, `DISABLE_PROFILER`, `DISABLE_DEBUGGER`) from appropriate YAML config files.
+- [**Integrating with Memorystore (redis)**](components/memorystore)
+  - The default Online Boutique deployment uses the in-cluster `redis` database for storing the contents of its shopping cart. The Memorystore deployment variation overrides the default database with its own Memorystore (redis) database. These changes directly affect `cartservice`.
+- [**Securing with Network Policies**](components/network-policies)
+  - Deploy fine granular `NetworkPolicies` for Online Boutique.
+- [**Creating Kubernetes Service Accounts**](components/service-accounts)
+  - Deploy fine granular `ServiceAccounts` for Online Boutique.
```

Comment:
**Praise:** The description for each variation is really well written. 👏 Concise. Thanks, Mathieu! Ahrar and I actually wanted to include a short description like this for each and every deployment variation. Currently, some of the non-Kustomize variations in `/README.md` don't contain good, short descriptions like this. We can think about that  _outside_ of this PR.